Tidy up links.md (#247)
carpentries-incubator/better-research-software: links.md

[biotools]: https://biotools.us/
[bitbucket]: https://bitbucket.org/

[carpentries-incubator]: https://carpentries-incubator.org/
[carpentries-lab]: https://carpentries-lab.org/
[carpentries-website]: https://carpentries.org/
[cff]: https://citation-file-format.github.io/
[cffinit-webapp]: https://bit.ly/cffinit
[choosealicense]: https://choosealicense.com/
[cldt-site]: https://carpentries.github.io/lesson-development-training/
[coc]: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
[codecheck]: https://codecheck.org.uk/project/
[codegen]: https://www.codegen.com/
[codemeta]: https://codemeta.github.io/
[coderefinery]: https://coderefinery.org/
[coderefinery-documentation]:https://coderefinery.github.io/documentation/
[coderefinery-testing]: https://coderefinery.github.io/testing/
[coderefinery-tools]: https://coderefinery.github.io/reproducible-research/
[computational-science]: https://en.wikipedia.org/wiki/Computational_science
[coursera-inline-comments]: https://www.coursera.org/tutorials/python-comment#inline-commenting-in-python
[cran]: https://cran.r-project.org/web/packages/

> 

Reviewer: ```suggestion

[diataxis-framework]: https://diataxis.fr/
```

Reviewer: see where I removed this on line 152

[dc]: https://datacarpentry.org/
[dc-lessons]: https://datacarpentry.org/lessons
[design-notes-template]: https://codimd.carpentries.org/HPwUE3FnTeSQJ9-_5EfU7Q?view#
[docker]: https://www.docker.com/
[dry-principle]: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

> [ds-testing]: https://ubc-dsci.github.io/reproducible-and-trustworthy-workflows-for-data-science/materials/lectures/06-intro-to-testing-code.html

Author: I'm guessing this page? https://ubc-dsci.github.io/reproducible-and-trustworthy-workflows-for-data-science/lectures/140-intro-to-testing-code.html

> [ds-testing]: https://ubc-dsci.github.io/reproducible-and-trustworthy-workflows-for-data-science/materials/lectures/06-intro-to-testing-code.html

Author: ```suggestion
[ds-testing]: https://ubc-dsci.github.io/reproducible-and-trustworthy-workflows-for-data-science/lectures/140-intro-to-testing-code.html
```

[swc-git-lesson-citation]: https://swcarpentry.github.io/git-novice/12-citation.html
[swc-git-lesson-licencing]: https://swcarpentry.github.io/git-novice/11-licensing.html
[swc-git-lesson-track]: https://swcarpentry.github.io/git-novice/04-changes.html
[swc-lessons]: https://software-carpentry.org/lessons/
[swhid]: https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html

[the-good-docs-project]: https://www.thegooddocsproject.dev/
[top-10-fair-things-per-domain]: https://librarycarpentry.org/Top-10-FAIR/
[ttw-guide-reproducible-research]: https://book.the-turing-way.org/reproducible-research/reproducible-research
[ttw-guide-version-control]: https://the-turing-way.netlify.app/reproducible-research/vcs
[ttw-handbook]: https://book.the-turing-way.org/
[ttw-illustrations]: https://doi.org/10.5281/zenodo.3332807
[ttw-reproducibility-def]: https://the-turing-way.netlify.app/reproducible-research/overview/overview-definitions
[turing-way-citation]: https://book.the-turing-way.org/communication/citable/citable-cff.html

[wiki-open-research-def]: https://en.wikipedia.org/wiki/Open_research
[wiki-reproducibility-def]: https://en.wikipedia.org/wiki/Reproducibility

> [write-the-docs]: https://www.writethedocs.org/[diataxis-framework]: https://diataxis.fr/

Reviewer: ```suggestion
[write-the-docs]: https://www.writethedocs.org/
```
